dev-python/pgspecial/pgspecial-1.11.7.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_5,3_6,3_7} )
+inherit distutils-r1
+
+DESCRIPTION="Python implementation of postgres meta commands"
+HOMEPAGE="https://github.com/dbcli/pgspecial"
+SRC_URI="https://github.com/dbcli/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/click-4.1[${PYTHON_USEDEP}]
+	>=dev-python/psycopg-2.7.4[${PYTHON_USEDEP}]
+	>=dev-python/python-sqlparse-0.1.19[${PYTHON_USEDEP}]
+"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
In EAPI-7 this is a BDEPEND, well, distutils already puts it in BDEPEND so you can remove this line.

Looks good otherwise!